src/config/index.js

```diff
@@ -159,6 +160,7 @@ let config = {
     hasWorkingCopySupport: false,
     maxUndoLevels: 200, // undo history size for the main form
     addonsInfo: addonsInfo,
+    useQuantaToolbar: true,
```

Comment:
I think we should rename this to `enableQuantaToolbar`.

Reply:
Done